src/clientAuth/bearerBody.js

```diff
@@ -8,12 +14,11 @@ class BearerBody extends ClientAuthnMethod {
   }
 
   /**
-   * More complicated logic then I would have liked it to be
-   *@param {*} cis Request class instance
-   *@param {*} ci Client information
-   *@param {*} requestArgs Request arguments
-   *@param {*} httpArgs HTTP header arguments
-   *@param {*} kwargs
+   *@param {ResourceRequest} cis Request class instance
+   *@param {ClientInfo} cliInfo Client information
+   *@param {Object.<string, string>} requestArgs Request arguments
```

Comment:
`{!Object<string, string>}`

Even better: provide the interface of the object here. No need to do in this CL though.